homeassistant/components/mqtt/alarm_control_panel.py

```diff
@@ -29,6 +30,8 @@
 _LOGGER = logging.getLogger(__name__)
 
 CONF_CODE_ARM_REQUIRED = 'code_arm_required'
+CONF_CODE_DISARM_REQUIRED = 'code_disarm_required'
```

Comment:
I don't think we need to add a `CONF_CODE_DISARM_REQUIRED` option; if a code is defined it is surely needed for disarming.

Reply:
I was adding it for consistency but I get your point. The conf_code_arm_required bypasses the built in comparator which I was trying to emulate for the disarm code.

If the code is been published should it bypass the code checks entirely?